ngraph/core/src/op/minimum.cpp

```diff
@@ -79,3 +79,19 @@ bool op::v1::Minimum::evaluate(const HostTensorVector& outputs,
     NGRAPH_OP_SCOPE(v1_Minimum_evaluate);
     return minimumop::evaluate_minimum(inputs[0], inputs[1], outputs[0], get_autob());
 }
+
+bool op::v1::Minimum::has_evaluate() const
+{
```

Comment:
Do we need to do this manually for all operations with `evaluate` supported?

Reply:
Yes, we need to do it manually, because each evaluate supports different cases.